<?xml version="1.0"?>
<launch>
  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
  <arg name="map_file" default="$(find navigationturt)/maps/map_20211018_125057.yaml"/>
  <arg name="x_pos" default="-3.0"/>
  <arg name="y_pos" default="1.0"/>
  <arg name="open_rviz" default="true"/>
  <arg name="move_forward_only" default="false"/>
  <arg name="gui" default="false"/>
  <arg name="node_start_delay" default="3.0"/>
  <arg name="objectives_file_location" default="$(find navigationturt)/scripts/objectives.txt"/>
  <!-- <arg name="objectives_file_location" default="/catkin_ws/src/navigationturt/scripts/objectives.txt"/> -->

   <include file="$(find navigationturt)/launch/godspeed.launch">
    <arg name="gui" value="$(arg gui)"/>
    <arg name="model" value="$(arg model)"/>
    <arg name="x_pos" value="$(arg x_pos)"/>
    <arg name="y_pos" value="$(arg y_pos)"/>
    <arg name="map_file"  value="$(arg map_file)"/>
    <arg name="open_rviz" value="$(arg open_rviz)"/>
    <arg name="move_forward_only" value="$(arg move_forward_only)"/>
    <arg name="node_start_delay" value="$(arg node_start_delay)"/>
    <arg name="exploration_mode" value="false"/>
    <arg name="teleoperation_mode" value="false"/>
    <arg name="uv_cleaning" value="false"/>
    <arg name="use_navigation_node" value="true"/>
    <arg name="use_objectives_file" value="true"/>
  </include>  

</launch>